pyqtgraph/graphicsItems/PColorMeshItem.py

```diff
@@ -200,7 +213,13 @@ def setData(self, *args):
 
             "ASCII from: <https://matplotlib.org/3.2.1/api/_as_gen/
                          matplotlib.pyplot.pcolormesh.html>".
+        autoLevels: bool, optional, default True
+            When set to True, PColorMeshITem will automatically select levels
+            based on the minimum and maximum values encountered in the data along the z axis.
+            The minimum and maximum levels are mapped to the lowest and highest colors 
+            in the colormap. The autoLevels parameter is ignored if ``enableAutoLevels==False`` 
```

Comment:
probably should change to

```
``enableAutoLevels is False``
```

last thing we should do is encourage doing equality operators on booleans :)  

(also a nit-pick level comment)

Reply:
Fixed